webknossos/webknossos/dataset/view.py

```diff
@@ -697,61 +710,72 @@ def some_work(args: Tuple[View, int], some_parameter: int) -> None:
         self,
-        work_on_chunk: Callable[[Tuple["View", "View", int]], None],
+        func_per_chunk: Callable[[Tuple["View", "View", int]], None],
         target_view: "View",
-        source_chunk_size: Vec3IntLike,  # in current mag
-        target_chunk_size: Vec3IntLike,  # in target view mag
+        source_chunk_size: Optional[Vec3IntLike] = None,  # in Mag(1)
+        target_chunk_size: Optional[Vec3IntLike] = None,  # in Mag(1)
         executor: Optional[
             Union[ClusterExecutor, cluster_tools.WrappedProcessPoolExecutor]
         ] = None,
         progress_desc: Optional[str] = None,
     ) -> None:
         """
-        This method is similar to 'for_each_chunk' in the sense, that it delegates work to smaller chunks.
+        This method is similar to `for_each_chunk` in the sense that it delegates work to smaller chunks,
+        given by `source_chunk_size` and `target_chunk_size` (both in Mag(1),
+        by default using the larger of the source_views and the target_views file-sizes).
         However, this method also takes another view as a parameter. Both views are chunked simultaneously
         and a matching pair of chunks is then passed to the function that shall be executed.
         This is useful if data from one view should be (transformed and) written to a different view,
         assuming that the transformation of the data can be handled on chunk-level.
-        Additionally to the two views, the counter 'i' is passed to the 'work_on_chunk',
-        which can be used for logging.
+        Additionally to the two views, the counter `i` is passed to the `func_per_chunk`, which can be used for logging.
+
         The mapping of chunks from the source view to the target is bijective.
-        The ratio between the size of the source_view (self) and the source_chunk_size must be equal to
-        the ratio between the target_view and the target_chunk_size. This guarantees that the number of chunks
-        in the source_view is equal to the number of chunks in the target_view.
-
-        Example use case: downsampling:
-        - size of source_view (Mag 1): (16384, 16384, 16384)
-        - size of target_view (Mag 2): (8192, 8192, 8192)
-        - source_chunk_size: (2048, 2048, 2048)
-        - target_chunk_size: (1024, 1024, 1024) // this must be a multiple of the file size on disk to avoid concurrent writes
+        The ratio between the size of the `source_view` (`self`) and the `source_chunk_size` must be equal to
+        the ratio between the `target_view` and the `target_chunk_size`. This guarantees that the number of chunks
+        in the `source_view` is equal to the number of chunks in the `target_view`.
+
+        Example use case: *downsampling*
+        - size of `source_view` (Mag 1): `(16384, 16384, 16384)`
+        - size of `target_view` (Mag 2): `(8192, 8192, 8192)`
+        - `source_chunk_size`: `(2048, 2048, 2048)`
+        - `target_chunk_size`: `(1024, 1024, 1024)`
```

Comment:
Since this example talks about downsampling, both chunk sizes would probably need to be equal to make sense.